cli/install/template.go

```diff
@@ -807,6 +822,8 @@ spec:
             path: /ready
             port: 9995
           failureThreshold: 7
+        securityContext:
+          runAsUser: 12102
```

Comment:
The proxy-injector pod listens on 443, and it fails to bind with this user ID set:

```
time="2018-12-06T01:28:39Z" level=fatal msg="listen tcp :443: bind: permission denied"
```

I'm not exactly sure how to fix that.

Reply:
Can we change the port that the service listens to? The container can still be on 443, but the service can be 8443 or something else?

Reply:
I changed the port and it looks like the proxy-injector pod start up now. Also, I verified that auto injection works on my minikube installation.